i18n/generate.py

```diff
@@ -175,6 +175,8 @@ def run(self, args):
         # Dummy text is not required. Don't raise exception if files are missing.
         for locale in config.CONFIGURATION.dummy_locales:
             merge_files(locale, fail_if_missing=False)
+        # Merge the source locale, so we have the canonical .po files.
+        merge_files(config.CONFIGURATION.source_locale, fail_if_missing=args.strict)
```

Comment:
I think that's probably unlikely, but it's good form to be explicit about it. I;ll add the test.